src/main/java/org/gwtproject/tools/cldr/DateTimeFormatInfoProcessor.java

```diff
@@ -35,627 +27,613 @@
-  }
-
-  private final RegionLanguageData regionLanguageData;
-
-  public DateTimeFormatInfoProcessor(File outputDir, InputFactory cldrFactory,
-      LocaleData localeData) {
-    super(outputDir, cldrFactory, localeData);
-    regionLanguageData = new RegionLanguageData(cldrFactory);
-  }
-
-  @Override
-  protected void cleanupData() {
-    System.out.println("Removing duplicates from date/time formats");
-    localeData.copyLocaleData("en", "default", "era-wide", "era-abbrev", "quarter-wide",
-        "quarter-abbrev", "day-wide", "day-sa-wide", "day-narrow", "day-sa-narrow", "day-abbrev",
-        "day-sa-abbrev", "month-wide", "month-sa-wide", "month-narrow", "month-sa-narrow",
-        "month-abbrev", "month-sa-abbrev");
-    removeUnusedFormats();
-    localeData.removeDuplicates("predef");
-    localeData.removeDuplicates("weekdata");
-    localeData.removeDuplicates("date");
-    localeData.removeDuplicates("time");
-    localeData.removeDuplicates("dateTime");
-    localeData.removeCompleteDuplicates("dayPeriod-abbrev");
-    computePeriodRedirects("day");
-    computePeriodRedirects("month");
-    computePeriodRedirects("day");
-    removePeriodDuplicates("day");
-    removePeriodDuplicates("month");
-    removePeriodDuplicates("quarter");
-    removePeriodDuplicates("era");
-  }
-
-  /**
-   * Generate an override for a method which takes String arguments, which
-   * simply redirect to another method based on a default value.
-   * 
-   * @param pw
-   * @param category
-   * @param locale
-   * @param method
-   * @param args
-   */
-  protected void generateArgMethod(PrintWriter pw, String category, GwtLocale locale,
-      String method, String... args) {
-    String value = localeData.getEntry(category, locale, "default");
-    if (value != null && value.length() > 0) {
-      pw.println();
-      if (getOverrides()) {
-        pw.println("  @Override");
-      }
-      pw.print("  public String " + method + "(");
-      String prefix = "";
-      for (String arg : args) {
-        pw.print(prefix + "String " + arg);
-        prefix = ", ";
-      }
-      pw.println(") {");
-      pw.print("    return " + method + Character.toTitleCase(value.charAt(0)) + value.substring(1)
-          + "(");
-      prefix = "";
-      for (String arg : args) {
-        pw.print(prefix + arg);
-        prefix = ", ";
-      }
-      pw.println(");");
-      pw.println("  }");
+
+    @Override
+    protected void cleanupData() {
+        System.out.println("Removing duplicates from date/time formats");
+        localeData.copyLocaleData("en", "default", "era-wide", "era-abbrev", "quarter-wide",
+                "quarter-abbrev", "day-wide", "day-sa-wide", "day-narrow", "day-sa-narrow", "day-abbrev",
+                "day-sa-abbrev", "month-wide", "month-sa-wide", "month-narrow", "month-sa-narrow",
+                "month-abbrev", "month-sa-abbrev");
+        removeUnusedFormats();
+//        localeData.removeDuplicates("predef");
```

Comment:
?

Reply:
those produced results where some locales didnt get the impls generated for one processor but where generated for another,
 so in LocaleInfo where you need both NumberConstants and DatetimeFormatInfo, one of those would be missing even though generating both for that locale is still valid.